lib/Tomcat/ServletTest.pm

```diff
@@ -53,7 +73,16 @@ sub request_information() {
 
 # test request header example
 sub request_header() {
-    assert_screen('tomcat-request-header-example', TIMEOUT);
+    my $mod_jk = shift;
```

Comment:
You could also use `my ($self, $mod_jk) = @_;` here.
I see that it is prominent in your code elsewhere, just for the sake of consistency